spec/features/file_distributor_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+describe "File Distributor#path" do
+  let (:pid) {"oregondigital:1234"}
```

Comment:
It might be a good idea to test more than one different non-alphanumeric character